modules/sno-du-configuring-ptp.adoc

```diff
@@ -108,7 +105,7 @@ spec:
         # Default interface options
         #
         clock_type OC
-        network_transport L2
+        network_transport UDPv4
```

Comment:
Any reason to change this? 

Looking at the source-cr for 4.11, it seems this should be L2.
https://github.com/openshift-kni/cnf-features-deploy/blob/release-4.11/ztp/source-crs/PtpConfigSlave.yaml#L99

Reply:
Updated in modules/snippets/ztp-ptp-config.adoc